src/test/java/com/puppycrawl/tools/checkstyle/meta/MetadataGeneratorUtilTest.java

```diff
@@ -54,6 +54,9 @@ public void testMetadataFilesGenerationAllFiles() throws Exception {
                         + "/tools/checkstyle/meta"))) {
             metaFiles = fileStream
                     .filter(Files::isRegularFile)
+                    .map(Path::toString)
+                    .filter(path -> !path.endsWith(".properties"))
```

Comment:
Why is this needed?

Reply:
```
System.getProperty("user.dir") + "/src/main/resources/com/puppycrawl/tools/checkstyle/meta")))
```
The Messages bundle is also present over there, so it is required to not process the `.properties` file

Reply:
Ok, but why don't we want it to process the properties?

Reply:
I am ok with everything else.

Since we split this PR up, I still don't see why this change is needed. Isn't it outside the scope of your changes since you no longer have any changes to the `MetadataGeneratorUtil`? Like above, what happens that is negative when this code is removed?

Reply:
This is included because what this test basically does is, count the number of metadata files and checks whether those are equal to the number of Checkstyle modules. Since all the metadata files are present in the same parent folder, I filtered out the `.properties` file so that it does not count them as metadata files. 

In a nutshell all metadata files are in `src/main/resources/com/puppycrawl/tools/checkstyle/meta` and the resource bundle is in the same file, we don't want `.properties` files to be counted as metadata files, hence I added this piece of code.